docs/Install.md

```diff
@@ -7,84 +7,79 @@
 If you want to customize the installation of the Splunk Operator, download a copy of the installation YAML locally, and open it in your favorite editor.
 
 ```
-wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-install.yaml
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
```

Comment:
Are we renaming the install.yaml file?

Reply:
changing back to operator/releases/download/1.0.5/splunk-operator-install.yaml

Reply:
fixed